css/style.css

```
@@ -545,43 +711,43 @@ img {
 }
 
 .main-footer {
-  background-color: var(--color-blue-3);
   color: var(--basic-white);
 }
 
 .footer-logo {
+  cursor: pointer;
   width: 220px;
-  min-height: 68px;
-  background-color: var(--basic-red);
-  box-shadow: inset 0 -5px 0 rgba(0, 0, 0, 0.24);
+  background: url(../img/logo-technomart.svg) var(--basic-red) no-repeat center;
+  height: 63px;
+  box-shadow: 0 5px 0 0 var(--logo-border);
+  background-size: 60%;
+}
+
+.footer-nav-list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
   display: flex;
-  justify-content: center;
+  flex-wrap: wrap;
 }
 
-.footer-phone {
-  color: inherit;
+.footer-nav-list:last-child {
+  margin: 33px 0 0 20px;
 }
 
-.footer-links-top {
-  width: 550px;
-  justify-self: end;
-  margin-top: 36px;
-  list-style: none;
-  padding: 0;
+.footer-nav-wrap ul li:not(:first-child) {
```

Нельзя ли сразу как-то клас кинуть на li?

---

Можно, просто жалко :) Учили-учили эти псевдоклассы, а не применяем(

---

```suggestion
.footer-top-links,
.footer-bottom-links {
  padding-left: 58px;
}

.footer-links-no-padding {
  padding: 0;
}
```

типа такого? Можно еще конечно опять через :not но уже по классу

---

Псевдоклассы ок  - плохо вложеность `.footer-nav-wrap ul li`